deployment/deploy_vrf_vms.py

```diff
@@ -130,7 +135,24 @@ async def deploy_vrf(
             environment={"PYTHONPATH": "/opt/packages"},
         )
 
+        if prepare_nodes:
+            failed_nodes, _ = await prepare_executor_nodes(executor_program_message.item_hash)
```

Comment:
The method warms up the executor function yes, and also return failed nodes. This is just launched the first time and any other one time that the user wants, because I added also the CLI command for that. This is why I called `prepare_executor_nodes`.